browser/ui/webui/help/brave_version_updater_mac_unittest.mm

```diff
@@ -25,6 +25,7 @@ bool UsesSparkle() {
 }
 
 TEST_F(BraveVersionUpdaterMacTest, UsesOmaha4WhenEnabled) {
-  scoped_feature_list_.InitAndEnableFeature(brave::kBraveUseOmaha4Alpha);
+  scoped_feature_list_.InitAndEnableFeature(
```

Comment:
tests shouldn't assume the default state for a feature so you should explicitly disable for `UsesSparkleByDefault`

Reply:
c629872e7b3767b61dec0c945215988e3eb750a8